packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -576,6 +577,18 @@ class Route extends EmberObject implements IRoute {
   /**
     This hook is executed when the router completely exits this route. It is
     not executed when the model for the route changes.
+    
+     ```app/routes/index.js
```

Comment:
This is a bit confusing, but there is the `deactivate` event, and the `deactivate` hook method. This sample is showing how to subscribe to the `on` event, but it is in the documentation for the hook method (see line 580).
Can you update the code sample?